testcases/healthcheck/healthcheck_kw.robot

```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Library           Collections
+Library           RequestsLibrary
+Library           String
+# For regular execution
+Library           KubeLibrary
+# For incluster execution
+#Library           KubeLibrary    None    True    False
+# For development
+#Library           ../../src/KubeLibrary/KubeLibrary.py  ~/.kube/k3d
+
+*** Keywords ***
+Healthcheck
+    ${output}=  Get Healthcheck
+	Log   \nHealthcheck ${output}:  console=True
```

Comment:
It would be nice if your example keyword could also verify if all checks have passed with ok.

Reply:
Evaluated the response and verified if all checks have passed  with ok.